gallery/gallery/lib/l10n/intl_en_US.arb

```diff
@@ -13,10 +13,100 @@
+  "@bannerDemoMultipleText": {
+    "description": "When the user clicks this button the Banner will toggle multiple actions or a single action"
+  },
+  "bannerDemoLeadingText": "Leading Icon",
+  "@bannerDemoLeadingText": {
+    "description": "If user clicks this button the leading icon in the Banner will disappear"
+   },
+  "dismiss": "DISMISS",
+  "@dismiss": {
+    "description": "When text is pressed the banner widget will be removed from the screen."
+  },
   "backToGallery": "Back to Gallery",
   "@backToGallery": {
     "description": "Semantic label for back button to exit a study and return to the gallery home page."
   },
+  "cardsDemoTappable": "Tappable",
+  "@cardsDemoTappable": {
+    "description": "The user can tap this button"
+  },
+  "cardsDemoSelectable": "Selectable (long press)",
+  "@cardsDemoSelectable": {
+    "description": "If the user taps and hold this card. The card will toggled between on and off."
+  },
+  "cardsDemoExplore": "Explore",
+  "@cardsDemoExplore": {
+    "description": "Click to see more about the content in the cards demo."
+  },
+  "cardsDemoExploreSemantics": "Explore {destinationName}",
+  "@cardsDemoExploreSemantics": {
+    "description": "Semantics label for Explore. Label tells user to explore the destinationName to the user. Example Explore Tamil",
+    "placeholders": {
+      "destinationName": "Tamil"
+    }
+  },
+  "cardsDemoShareSemantics": "Share {destinationName}",
+  "@cardsDemoShareSemantics": {
+    "description": "Semantics label for Share. Label tells user to share the destinationName to the user. Example Share Tamil",
+    "placeholders": {
+      "destinationName": "Tamil"
+    }
+  },
+  "cardsDemoTravelDestinationTitle1": "Top 10 Cities to Visit in Tamil Nadu",
+  "@cardsDemoTravelDestinationTitle1": {
+    "description": "The top 10 cities that you can visit in Tamil Nadu"
+  },
+  "cardsDemoTravelDestinationDescription1": "Number 10",
+  "@cardsDemoTravelDestinationDescription1": {
+    "description": "Number 10"
+  },
+  "cardsDemoTravelDestinationCity1": "Thanjavur",
+  "@cardsDemoTravelDestinationCity1": {
+    "description": "Thanjavur the city"
+  },
+  "cardsDemoTravelDestinationLocation1": "Thanjavur, Tamil Nadu",
+  "@cardsDemoTravelDestinationLocation1": {
+    "description": "Thanjavur, Tamil Nadu"
+  },
+  "cardsDemoTravelDestinationTitle2": "Artisans of Southern India",
+  "@cardsDemoTravelDestinationTitle2": {
+    "description": "Artist that are from Southern India"
+  },
+  "cardsDemoTravelDestinationDescription2": "Silk Spinners",
+  "@cardsDemoTravelDestinationDescription2": {
+    "description": "Silk Spinners"
+  },
+  "cardsDemoTravelDestinationCity2": "Chettinad",
+  "@cardsDemoTravelDestinationCity2": {
+    "description": "Chettinad the city"
+  },
+  "cardsDemoTravelDestinationLocation2": "Sivaganga, Tamil Nadu",
+  "@cardsDemoTravelDestinationLocation2": {
+    "description": "Sivaganga, Tamil Nadu"
```

Comment:
Specify that this is referring to a location.

Reply:
Done